appveyor.yml

```diff
@@ -1,3 +1,5 @@
+image:
+  - Visual Studio 2015
```

Comment:
I've had to add this to have appveyor use the correct image. Not sure if this is an appveyor setting depending on the account that's running it ...

Reply:
Totally agree, thanks